sql/moz-fx-data-marketing-prod/ga_derived/downloads_with_attribution_v2/query.sql

```diff
@@ -0,0 +1,367 @@
+-- Query for ga_derived.downloads_with_attribution_v2
+CREATE TEMP FUNCTION normalize_browser(browser STRING) AS (
+  CASE
+    WHEN `moz-fx-data-shared-prod.udf.ga_is_mozilla_browser`(browser)
+      THEN 'Firefox'
+    WHEN browser IN ('Internet Explorer')
+      THEN 'MSIE'
+    WHEN browser IN ('Edge')
+      THEN 'Edge'
+    WHEN browser IN ('Chrome')
+      THEN 'Chrome'
+    WHEN browser IN ('Safari')
+      THEN 'Safari'
+    WHEN browser IN ('(not set)')
+      THEN NULL
+    WHEN browser IS NULL
+      THEN NULL
+    ELSE 'Other'
+  END
+);
+
+WITH all_hits AS (
+  SELECT
+    clientId AS client_id,
+    visitId AS visit_id,
+    hit.appInfo.landingScreenName AS landing_page,
+    hit.page.pagePath AS pagepath,
+    CASE
+      WHEN (
+          hit.type = 'EVENT'
+          AND hit.eventInfo.eventAction = 'Firefox Download'
+          AND hit.eventInfo.eventCategory IS NOT NULL
+          AND hit.eventInfo.eventLabel LIKE 'Firefox for Desktop%'
+        )
+        THEN TRUE
+      ELSE FALSE
+    END AS has_ga_download_event,
+    hit.type AS hit_type,
+    CASE
+      WHEN (hit.type = 'EVENT' AND hit.eventInfo.eventAction = 'Stub Session ID')
+        THEN hit.eventInfo.eventLabel
+      ELSE NULL
+    END AS download_session_id
+  FROM
+    `moz-fx-data-marketing-prod.65789850.ga_sessions_*` AS ga
+  CROSS JOIN
+    UNNEST(hits) AS hit
+  WHERE
+    _TABLE_SUFFIX
+    BETWEEN FORMAT_DATE('%Y%m%d', DATE_SUB(@download_date, INTERVAL 2 DAY))
+    AND FORMAT_DATE('%Y%m%d', DATE_ADD(@download_date, INTERVAL 1 DAY))
```

Comment:
Does this indicates that _no_ ga hits come in on the same day as the `session_id` presents itself in the stub attribution service logs?